command/pr_test.go

```diff
@@ -278,17 +278,14 @@ func TestPRList(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	eq(t, output.Stderr(), `
-Showing 3 of 3 pull requests in OWNER/REPO
```

Comment:
I would recommend that we hold off merging this until we figure out how to simulate tty mode for tests so we can keep asserting this output. The header is an important part of output and I wish we can keep test coverage of it.

Reply:
I think that is fair. In reviewing #572 and then returning to this I think we need to step back and design a holistic approach to determining what kind of output (both formatting and color) to use; something routed via the `cmd` reference that can be stubbed appropriately when running commands in a test.

Reply:
Samah